advocacy_docs/partner_docs/CohesityDataProtectforPostgreSQL/04-ConfiguringCohesityDataProtectforPostgreSQL.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Configuring Cohesity DataProtect for PostgreSQL'
+description: 'Walkthrough on configuring Cohesity DataProtect for PostgreSQL'
+---
+
+Implementing Cohesity DataProtect for PostgreSQL with EDB Postgres Advanced Server requires the following components:
+
+- EDB Postgres Advanced Server 
+- Cohesity Backup Appliance
+- Cohesity Linux Agent 
+- Cohesity PostgreSQL Agent 
+
+### Prerequisites
+
+- A running EDB Postgres Advanced Server instance
+- Cohesity Backup Appliance installed and running on VMware
+- Cohesity agents installed on the same machine as your EDB Postgres Advanced Server database
```

Comment:
should just name the agents and not say agents.   So Linux and PostgreSQL agentes.